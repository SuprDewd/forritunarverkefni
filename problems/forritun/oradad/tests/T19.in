-79 -77 -69
