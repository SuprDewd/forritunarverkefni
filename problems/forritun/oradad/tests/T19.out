-77 -79 -69
